vmm/Cargo.toml

```diff
@@ -33,4 +33,3 @@ tempfile = ">=3.0.2"
 
 [features]
 vsock = ["devices/vsock"]
-
```

Comment:
nit: can you leave this line as it was?

Reply:
Trim whitespace on save strikes again. Fixed in latest commit, along with signature.